lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -274,11 +268,19 @@ private function hydrateReferenceField(
                 $this->getDefaultValue($fieldMapping)
             );
 
-            if ($referencedNode) {
-                $accessor->set(
+            try {
+                if ($referencedNode) {
+                    $accessor->set(
                     $fieldName,
-                    $this->proxyFactory->createProxyForNode($document, $referencedNode, $options)
+                    $proxyFactory->createProxyForNode($document, $referencedNode, $options)
                 );
+                }
+            } catch (\Exception $e) {
```

Comment:
By catching the all exceptions we get to see exactly `which field` has the error - which is a crucial piece of information, if we do not catch, then we lose this information.